src/styles/blocks/card.scss

```
@@ -0,0 +1,101 @@
+main {
+  @extend %flex-center;
+}
+
+.catalog-wrapper {
+  @extend %flex-space-between;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  padding: 50px 40px;
+  gap: 48px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 14px;
+  width: 200px;
+  background: $main-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 2px;
```

```suggestion
  border: 1px solid #f3f3f3;
  border-radius: 5px;
```

Your `card` is a BEM block. It cannot has outer-spacings (margins)